megamek/data/mechfiles/vehicles/3039u/Coolant Truck (Hover).blk

```diff
@@ -37,11 +41,7 @@ Hover
 </motion_type>
 
 <transporters>
```

Comment:
I read this as a botched correction, not as an attempt to create two separate bays.  Updating to a single bay with the new size produced entries that match the TRO.